addons/website_slides/views/slide_channel_partner_views.xml

```diff
@@ -24,6 +24,7 @@
             <field name="arch" type="xml">
                 <tree string="Attendees" editable="top" sample="1">
                     <field name="channel_id" string="Course Name"/>
+                    <field name="partner_id" string="Partner"/>
```

Comment:
the `string="Partner"` doesn't seem necessary, it's already Partner if I don't set it

side note: if we add `string="Something"` (but it doesn't seem necessary here) we would also need to add an entry reference to the string in `addons/website_slides/i18n/website_slides.pot` (following [Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions](https://github.com/odoo/odoo/wiki/Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions)) so it is available to be translated in other language (or it would be only available in english)

Reply:
done